Stack/Opc.Ua.Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -309,6 +294,43 @@ public virtual void Validate(X509Certificate2Collection chain)
             }
         }
 
+        /// <summary>
+        /// recursively checks whether any of the service results or inner service results
+        /// of the input sr must not be suppressed.
+        /// The list of supressible status codes is - for backwards compatibiliyt - longer
+        /// than the spec would imply.
+        /// (BadCertificateUntrusted and BadCertificateChainIncomplete
+        /// must not be supressed according to (e.g.) version 1.04 of the spec)
+        /// </summary>
+        /// <param name="sr"></param>
+        /// <returns></returns>
+        private bool containsUnsuppressibleSC(ServiceResult sr)
```

Comment:
According to naming rules this should be in upper case, i.e. "**C**ontainsUnsuppressibleSC"

Reply:
It would be better to put this method in ServiceResult.cs as "`public static bool IsSuppressible(ServiceResult status)`",
the same way as `ServiceResult.IsGood()/ .IsBad() IsUncertain()` ...

https://github.com/OPCFoundation/UA-.NETStandard/blob/f904139298a052048655d24f569a3e8efaf1a60d/Stack/Opc.Ua.Core/Types/Utils/ServiceResult.cs#L549-L557

Reply:
I think the name IsSuppressable is good but I think it can stay in the CertValidator file, is this is really a very specific function tied into cert validation. while the IsBad etc. is more a generic feature used in many places.